payments-core/src/main/java/com/stripe/android/paymentsheet/model/SupportedSavedPaymentMethod.kt

```diff
@@ -0,0 +1,71 @@
+package com.stripe.android.paymentsheet.model
+
+import android.content.res.Resources
+import androidx.annotation.DrawableRes
+import com.stripe.android.R
+import com.stripe.android.model.CardBrand
+import com.stripe.android.model.PaymentMethod
+
+/**
+ * Enum defining all types of saved payment method supported on Payment Sheet.
+ *
+ * These are fetched from the
+ * [PaymentMethods API endpoint](https://stripe.com/docs/api/payment_methods/list) for returning
+ * customers.
+ */
+internal enum class SupportedSavedPaymentMethod(
+    val type: PaymentMethod.Type
+) {
+    Card(PaymentMethod.Type.Card),
+    SepaDebit(PaymentMethod.Type.SepaDebit);
+
+    companion object {
+        internal fun fromCode(code: String) =
+            values().firstOrNull { it.type.code == code }
+
+        internal fun isValid(paymentMethod: PaymentMethod) =
+            when (paymentMethod.type) {
+                PaymentMethod.Type.Card -> paymentMethod.card != null
+                PaymentMethod.Type.SepaDebit -> paymentMethod.sepaDebit != null
+                else -> false
+            }
+    }
+}
+
+@DrawableRes
+internal fun PaymentMethod.getSavedPaymentMethodIcon(): Int? = when (type) {
```

Comment:
probably move these to dedicate `PaymentMethodKtx` and `CardBrandKtx` files

Reply:
Is there any pattern for using `*Ktx` file name? Moved them to `PaymentMethodsUiExtension`

Reply:
we have been adding extensions to classes with classNameKtx, like [StripeIntentKtx](https://github.com/stripe/stripe-android/blob/c294238538f0a6a1601228c263a78bf7b110adb8/payments-core/src/main/java/com/stripe/android/model/StripeIntentKtx.kt#L6-L5) and [StripeKtx](https://github.com/stripe/stripe-android/blob/ad426ca33105e7bc5f17c027f8146354d13f75ae/payments-core/src/main/java/com/stripe/android/StripeKtx.kt#L49)